.gitignore

```diff
@@ -6,3 +6,5 @@ build
 .coverage*
 .idea/*
 /dist
+**/*.sqlite
+pycytominer/tests/test_data/collate/backend/2021_04_20_Target2/BR00121431/BR00121431.csv
```

Comment:
Why is this ignored? Isn't it now part of the test suite?

Reply:
Changed this to make more clear